site/docs-md/guides/push-notifications-firebase.md

```diff
@@ -4,11 +4,12 @@ description: Learn how to get Firebase Cloud Messaging working on iOS and Androi
 url: /docs/guides/push-notifications-firebase
 contributors:
   - bryplano
+  - javebratt
 ---
 
 # Using Push Notifications with Firebase in an Ionic + Angular App
 
-**Web Framework**: Ionic 4 + Angular  
+**Web Framework**: Ionic 5 + Angular
```

Comment:
we actually want to get away from this. Couple reasons, because as of Ionic 4, "it's just Ionic", and like in the Deep Links guide, we (ideally) will make it open to all frameworks, have examples as separate headers (Angular, React, etc).

Keep the Platforms one though.

Reply:
I left it as Angular.

I was gonna remove it all together, but there's Angular specific code for the Capacitor push plugin usage.